test/tutorial/scripts/cli/put_delete_get_sub_cli.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Creates a sub based given a replica and a url
-instance_info=$(hca dss put-subscription --callback-url https://www.example.com --replica aws) 
+instance_info=$(hca dss put-subscription --callback-url https://dcp-cli-tutorials-put-get-delete-sub.com --replica aws) 
```

Comment:
nit. we should use `*.humancellatlas.org` for the URLs.

Reply:
Resolved